modules/tinymce/src/themes/silver/main/ts/ui/window/SilverDialog.ts

```diff
@@ -40,9 +40,9 @@ const renderDialog = <T extends Dialog.DialogData>(dialogInit: DialogManager.Dia
 
   const objOfCells = SilverDialogCommon.extractCellsToObject(storedMenuButtons);
 
-  const footer = renderModalFooter({
-    buttons: storedMenuButtons
-  }, dialogId, backstage);
+  const footer = storedMenuButtons.length === 0
+    ? Optional.none()
+    : Optional.some(renderModalFooter({ buttons: storedMenuButtons }, dialogId, backstage));
```

Comment:
```suggestion
  const footer = Optional.from(renderModalFooter({ buttons: storedMenuButtons }, dialogId, backstage)).filter(() => storedMenuButtons.length !== 0);
```
Just a different method for defining optionals that I've been implementing, no difference in outcome in this case.

Reply:
Using `Optionals.someIf` probably would be the cleanest approach. In any case, it is more efficient to do an `if` check than creating the component then filtering it afterwards

Reply:
That's true, this is just useful in some cases where the `some` value could potentially be null or undefined as well, so I agree it's unnecessary here and just a suggestion.

Reply:
Thanks for the suggestions, implemented `Optionals.someIf`.